Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG BASE_IMAGE=gcr.io/distroless/python3-debian12:nonroot
+
```

Comment:
Can we move after the first comment block before "# This Dockerfile is a multi-stage build. The first stage builds the frontend."?